livnote/frontend/desktop/state/data.svelte.ts

```
@@ -27,16 +27,38 @@ class DataState {
   userDetails = $state<UserDetails | null>(null)
+  currentNoteId = $state<string | null>(null);
   private _unlisteners: Array<() => void> = [];
   sharedUsers = $state([]);
   collaborators = $state<Collaborator[]>([]);
+  clientId: number = 0;
+  currentNoteTitle = $state<string>("");
 
+  getNotesCoordinator(): NotesCoordinator | null {
+    return this.notesCoordinator;
+  }
+  getCurrentNoteId(): String | null {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)